jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/MavenProjectPropertiesTest.java

```diff
@@ -168,6 +176,29 @@ private static Path getResource(String path) throws URISyntaxException {
     return Paths.get(Resources.getResource(path).toURI());
   }
 
+  private static Path zipUpDirectory(Path sourceRoot, Path targetZip) throws IOException {
+    try (ZipOutputStream zipOut = new ZipOutputStream(Files.newOutputStream(targetZip));
+        Stream<Path> stream = Files.walk(sourceRoot)) {
```

Comment:
Oh sorry almost forgot, I don't think we need this `Stream<Path> stream = Files.walk(sourceRoot)` anymore.

Reply:
Nice catch. Not feeling well today. Should probably go home early.